pkg/controller/windowsmachine/windows/windows.go

```diff
@@ -200,13 +202,13 @@ func (vm *windows) ConfigureHybridOverlay(nodeName string) error {
 
 func (vm *windows) ConfigureCNI(configFile string) error {
 	// copy the CNI config file to the Windows VM
-	if err := vm.CopyFile(configFile, cniDir); err != nil {
-		return errors.Errorf("unable to copy CNI file %s to %s", configFile, cniDir)
+	if err := vm.CopyFile(configFile, cniConfDir); err != nil {
+		return errors.Errorf("unable to copy CNI file %s to %s", configFile, cniConfDir)
 	}
 
-	cniConfigDest := cniDir + filepath.Base(configFile)
+	cniConfigDest := cniConfDir + filepath.Base(configFile)
 	// run the configure-cni command on the Windows VM
-	configureCNICmd := remoteDir + "wmcb.exe configure-cni --cni-dir=\"" +
+	configureCNICmd := k8sDir + "wmcb.exe configure-cni --install-dir=\"" + cniDir + "--cni-dir=\"" +
```

Comment:
Why is the `install-dir` change necessary? If you are changing it why not explicitly change it to C:\k

Reply:
install-dir defaults to c:\K, wanted to make sure the directory is c:\k\cni for all cni related downloads.

Reply:
Ah ok, i misread the change

I feel like this is misusing the `install-dir` option
The option is documented as: `// installDir is the main installation directory` 
https://github.com/openshift/windows-machine-config-bootstrapper/blob/master/cmd/bootstrapper/configure_cni.go#L38
and c:\k\cni is not the main installation directory. 

Both install-dir and cni-dir are being set to the same folder here which I don't think we want. install-dir should be consistent throughout all invocations of wmcb.exe, so that we can reference files that are expected to be there after `wmcb.exe initialize-kubelet` is ran.
See here where the kubelet binary is set in relation to the install-dir? https://github.com/openshift/windows-machine-config-bootstrapper/blob/master/pkg/bootstrapper/bootstrapper.go#L158

Have you tested this and does this work? I dont think CI is passing because of this.

Reply:
I think I missed the comment for "main" installation directory, in this case the install-dir option will no longer be required as it defaults to c:\k. I had tested it manually, and the CI logs show the tests timed out in East-West networking, but I will make this change and retest.

Reply:
I am retrieving the install-dir option and directing it to C:\k as not doing so is also copying the cni config file to C:\k which is not desirable and we want it to be only stored in C:\Temp\cni